spec/components/validation/Validation.spec.tsx

```diff
@@ -198,7 +198,7 @@ describe('Validation Component', () => {
       await validator;
       await validate;
       wrapper.render();
-      expect(wrapper.find('.tk-validation__errors').text()).toEqual('Required');
+      expect(wrapper.find('.tk-validation__errors').text()).toContain('Required');
```

Comment:
any reason for changing this?

Reply:
Because we have the new little cute icon ! 
![image](https://user-images.githubusercontent.com/56824367/146236833-6f2c9963-70c5-4a0d-85ad-ac3e078bfd83.png)
Therefore, the content is not "equal" to the string, but "contain" the string